local.properties

```diff
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Sun Jun 28 10:52:25 CEST 2020
-sdk.dir=C\:\\Users\\kage\\AppData\\Local\\Android\\Sdk
+#Mon Jun 29 19:15:01 CEST 2020
+sdk.dir=C\:\\Users\\Anon\\AppData\\Local\\Android\\Sdk
```

Comment:
This file illustrated why it is necessary to ignore some (auto generated) files. This path is user specific and would change every time with every commit of another developer.

Auto-generated files as well as builds and user specific files are usually ignored from Git.